src/components/App.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+export default class App extends React.Component {
```

Comment:
Are we able to keep this a functional component?

Reply:
@irmerk I am porting the code to use functional components.